.vscode/settings.json

```diff
@@ -1,10 +0,0 @@
-{
```

Comment:
I think we should keep this file as it provides a schema for the test files.

Reply:
This file is in the `.gitignore`, so I don't think it should have been commited in the first place.

If you think we should have the file, then it would make sense to also update gitignore.

Reply:
It's in the gitignore since vscode frequently edits the file. So we don't want to commit local user edits that vscode automatically does.

Reply:
Makes sense. I've added it back.